ui/src/lib/store/canvasSlice.tsx

```diff
@@ -142,7 +131,11 @@ function createNewNode(type: "SCOPE" | "CODE" | "RICH", position): Node {
           // the top-left corner (the reason is unknown). Thus, we have to
           // specify the height here. Note that this height is a dummy value;
           // the content height will still be adjusted based on content height.
-          height: 200,
+          //
+          // NOTE for import ipynb: we need to specify some reasonable height so that
+          // the imported pods can be properly laid-out. 130 is a good one.
+          // This number MUST match the number in Canvas.tsx (refer to "A BIG HACK" in Canvas.tsx).
+          height: 130,
```

Comment:
This magic number `130` must be kept consistent with the `130` in `Canvas.tsx`. Let's store it in one global variable and access it in both files. Also, we can use some floating number to mark it special, e.g., `136.66`.

Reply:
Given it's a hacky way to apply autolayout, I think we can use 130 here. We need to find a better way for autolayer eventually.

Reply:
> Given it's a hacky way

Precisely because it is hacky, we want to avoid breaking it accidentally, by:
- using a global variable to avoid potential inconsistency error
- using some floating numbers to prevent a pod from being the same value.

Reply:
I see you have changed to a global variable. The floating number isn't that critical (as long as it's not 120).

Reply:
SG, but let me send a quick patch to change the height to floating number.